parcels/examples/example_stommel.py

```diff
@@ -122,14 +125,15 @@ class MyParticle(ParticleClass):
     return pset
 
 
+@pytest.mark.parametrize('pset_mode', ['soa', 'aos'])
```

Comment:
Not sure if this is possible in pytest, but for easy further extension with other datastructures, would it be an idea to change these lines to e.g. `@pytest.mark.parametrize('pset_mode', pset_types)` and then define `pset_types = ['soa', 'aos']` somewhere at the top of this (and other) test files? If we then add another ParticleSet type, we only need to change one line?

Reply:
hm, yes, that could work, nice suggestion.

Reply:
@erikvansebille have a look at the new edits, see if that is what you had in mind, please.